packages/template-retail-react-app/app/pages/product-list/partials/color-refinements.jsx

```diff
@@ -46,6 +48,13 @@ const ColorRefinements = ({filter, toggleFilter, selectedFilters}) => {
                                     variant="outline"
                                     marginRight={0}
                                     marginBottom="-1px"
+                                    aria-label={intl.formatMessage(
+                                        {
+                                            id: 'colorRefinements.label.hitCount',
+                                            defaultMessage: '{colorLabel} {colorHitCount}'
```

Comment:
```suggestion
                                            defaultMessage: '{colorLabel} ({colorHitCount})'
```
To match the previous text.